test/lcov_ex_test.exs

```diff
@@ -6,7 +6,13 @@ defmodule LcovExTest do
     setup do
       mix_path = "#{File.cwd!()}/example_project/mix.exs" |> String.replace("//", "/")
       MixFileHelper.backup(mix_path)
-      config = [test_coverage: [tool: LcovEx, ignore_paths: ["deps/"]]]
+      config = [
+        test_coverage: [
+          tool: LcovEx,
+          ignore_paths: ["deps/"],
+          ignore_modules: [ExampleProject.ExampleIgnoreModule, ~r/.*ExampleIgnoreRegex.*/]
```

Comment:
I would prefer to do a separate test for the new feature, and in that one assert that the module **is not** in the coverage file (instead of testing the omission in the print).